### ISSUE TITLE
schema is trying to validate disabled nodes

### ISSUE THREAD

**Issue by mripard** [robherring/dt-schema#8]
In a number of occassions, a device tree node can only be complete when a board DTS enables it.

For example, a DT binding of a network MAC might require a PHY, expressed through a phandle using a phy property. Yet, there's no way to fill that phy property on devices that wouldn't enable that node, since it's very likely that the board doesn't make any use of that controller and therefore doesn't have any PHY.

This current generates a warning that shouldn't be there. Maybe we should just stop validating the nodes that have a status = "disabled" property?

I've looked around but couldn't really find a way to do this in the tooling.

**Comment by AlexanderStein** [comment]
I don't think skipping disabled node validation is the correct way. In the given case network MAC, where are still properties which can be validates, even if there is no PHY.
Just ignoring missing properties seem the way to go.

TLDR: This should be addressed by https://github.com/robherring/dt-schema/commit/5466428c38914fe3a0db212cab16a671b2d69506 already
